### PR TITLE
add 'bytes' to aligned/alive_total for clarity

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
distinguish between '# bytes' and '# accounts' with better struct member names

#### Summary of Changes

Fixes #

### CODE REVIEW

File: runtime/src/ancient_append_vecs.rs [hunk 1]
```diff
@@ -49,7 +49,6 @@ impl<'a> AccountsToStore<'a> {
             if available_bytes >= account_size {
                 available_bytes = available_bytes.saturating_sub(account_size);
             } else if index_first_item_overflow == num_accounts {
-                available_bytes = 0;
```
Comment (apfitzge) [inline runtime/src/ancient_append_vecs.rs:52]: is this removal intentional? everything else was just a rename, this seems to have a functional difference.

Comment (jeffwashington) [reply runtime/src/ancient_append_vecs.rs]: no, that was left over from a previous change. Looks like it would have had no ill effect, but still incorrect.

**Review by apfitzge — APPROVED** [review]
Lgtm
